torch/csrc/jit/passes/graph_rewrite_helper.cpp

```diff
@@ -34,7 +34,7 @@ c10::optional<IValue> getIValue(
   return toIValue(getValue(name, match_vmap, vmap));
 }
 
-void replaceConvolutionWithConv2d(std::shared_ptr<Graph>& graph) {
+void replaceConvolutionWithAtenConv(std::shared_ptr<Graph>& graph) {
```

Comment:
@jerryzh168 @raghuramank100 From the aten op looks like the main distinguishing difference between aten::conv1d and aten::conv2d is the output_padding {0} vs {0,0} https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Convolution.cpp#L535-L543

Reply:
In addition, we should check for bool transposed along with output_padding to map to conv1d/2d/3d and the transpose ops.

Reply:
We're already checking `transposed_value` below. When we add deconv here we can support that too.

Reply:
I think the length of stride/padding/dilation should be different as well, it makes more sense to check these arguments that are actually meaningful I think